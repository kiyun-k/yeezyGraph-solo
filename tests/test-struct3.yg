struct N {
	int i;
	string j;
}

int main(){
	struct N n;
	n~i = 15;
	print(n~i);
	n~j = "foo";
	n~i = 16;
	print(n~i);
	prints(n~j);
	return 0;
}