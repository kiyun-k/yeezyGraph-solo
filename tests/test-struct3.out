15
16
foo